metricbeat/module/postgresql/statements/_meta/data.json

```diff
@@ -0,0 +1,54 @@
+{
+    "@timestamp": "2017-10-12T08:05:34.853Z",
+    "beat": {
+        "hostname": "host.example.com",
+        "name": "host.example.com"
+    },
+    "metricset": {
+        "host": "postgresql:5432",
+        "module": "postgresql",
+        "name": "statements",
+        "rtt": 115
+    },
+    "postgresql": {
+        "statements": {
```

Comment:
Should the metricset be called `statement`? Because each even only contains one statement.

Reply:
I'd agree in general with that, but if we keep it as `statements` then all metricsets follow the exact naming of pg stats tables (`pg_stat_activity`, `pg_stat_database`, `pg_stat_bgwriter` and `pg_stat_statements`).

Reply:
In general I would prefer the the naming convention superseed service specific namings: https://www.elastic.co/guide/en/beats/devguide/current/event-conventions.html This provides a unified querying experience on the consumer side.

Reply:
Ok, go for `statement` then.